lib/libc/stdio/vfprintf.c

```diff
@@ -862,18 +862,16 @@ reswitch:	switch (ch) {
 			 *	-- ANSI X3J11
 			 */
 #if __has_feature(capabilities)
-#ifdef __CHERI_PURE_CAPABILITY__
-			cap = GETARG(void *);
-			ujval = cheri_getaddress(cap);
-#else
-			if (flags & LONGINT) {
-				cap = *GETARG(void * __capability *);
-				ujval = cheri_getaddress(cap);
-			} else {
+#ifndef __CHERI_PURE_CAPABILITY__
+			if (!(flags & LONGINT)) {
```

Comment:
```suggestion
			if ((flags & LONGINT) == 0) {
```
I personally find the == 0 version easier to read, but there seems to be a fairly even split across the source tree for both styles, so I guess it's whatever you prefer.
`!\([_\w\d]+ & [_\w\d]+\)` -> 2265 times
`\([_\w\d]+ & [_\w\d]+\) == 0` -> 3427 times

Reply:
I was matching style in this file.